client/www/scripts/lib/jquery-from-global.js

```diff
@@ -0,0 +1 @@
+module.exports = (window || global).jQuery;
```

Comment:
is this just including the library or exposing the 'jQuery' namespace globally?  

i.e do we need to take into account '$' alias?